orchestration/dagster_orchestration/hca_manage/snapshot.py

```diff
@@ -2,15 +2,20 @@
 from dataclasses import dataclass, field
 from datetime import datetime, date
 import logging
+import sys
 from typing import Optional
 
 from data_repo_client import RepositoryApi, SnapshotRequestModel, SnapshotRequestContentsModel, EnumerateSnapshotModel
+from dagster_utils.contrib.data_repo.jobs import poll_job, JobPollException
 from dagster_utils.contrib.data_repo.typing import JobId
 
 from hca_manage import __version__ as hca_manage_version
 from hca_manage.common import data_repo_host, data_repo_profile_ids, DefaultHelpParser, get_api_client, \
     query_yes_no, tdr_operation, setup_cli_logging_format
 
+MAX_SNAPSHOT_CREATE_POLL_SECONDS = 120
```

Comment:
⛏️ This is the timeout for deletion, right? 
```suggestion
MAX_SNAPSHOT_DELETE_POLL_SECONDS = 120
```